client/app/bundles/course/assessment/submission/submit-answer.js

```diff
@@ -87,7 +87,7 @@ function onAnswerSubmit(e) {
 
 // Find all the spinning jobs in the page and check their statuses.
 function checkSubmittedJobs() {
-  $('a.btn.submitted').each(function (index) {
+  $('a.btn.submitted').each(function checkJob(index) {
```

Comment:
Actually might be better to config and ignore the lint, sometimes forcing a name doesn't really make sense.

like the following case giving a name leads to some duplications:
```
$('xxx).onChange(function onChange() {
});
```

Thoughts ?